## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 457580`
Commit: [`1e0dbfb1c0d206575fa4bf7773913f1806a8f4f0`](https://github.com/NixOS/nixpkgs/commit/1e0dbfb1c0d206575fa4bf7773913f1806a8f4f0) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/1e0dbfb1c0d206575fa4bf7773913f1806a8f4f0..pull/457580/head))
Merge: [`b581be7dba88478becb773fb7dd220ac47079e6a`](https://github.com/NixOS/nixpkgs/commit/b581be7dba88478becb773fb7dd220ac47079e6a)

Logs: https://github.com/mdaniels5757/nixpkgs-review-gha/actions/runs/18999888777


---
### `x86_64-linux`
<details>
  <summary>:x: 10 packages failed to build:</summary>
  <ul>
    <li>atomic-operator</li>
    <li>atomic-operator.dist</li>
    <li>knowsmore</li>
    <li>knowsmore.dist</li>
    <li>netexec</li>
    <li>netexec.dist</li>
    <li>python312Packages.psrpcore</li>
    <li>python312Packages.psrpcore.dist</li>
    <li>python312Packages.pypsrp</li>
    <li>python312Packages.pypsrp.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 8 packages built:</summary>
  <ul>
    <li>ilspycmd</li>
    <li>powershell</li>
    <li>powershell-editor-services</li>
    <li>python313Packages.psrpcore</li>
    <li>python313Packages.psrpcore.dist</li>
    <li>python313Packages.pypsrp</li>
    <li>python313Packages.pypsrp.dist</li>
    <li>windmill</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>python312Packages.psrpcore</summary>
<pre>      self.run()
    File &quot;/build/source/tests/test_integration_server.py&quot;, line 67, in run
      e = self.client.next_event()
          ^^^^^^^^^^^^^^^^^^^^^^^^
    File &quot;/build/source/tests/conftest.py&quot;, line 91, in next_event
      payload = self.next_payload()
                ^^^^^^^^^^^^^^^^^^^
    File &quot;/build/source/tests/conftest.py&quot;, line 79, in next_payload
      payload = self._incoming.get(timeout=60)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    File &quot;/nix/store/jd20rkmqmkfkcvk2wl2lmzz7acq4svlr-python3-3.12.12/lib/python3.12/queue.py&quot;, line 179, in get
      raise Empty
  _queue.Empty
  
    warnings.warn(pytest.PytestUnhandledThreadExceptionWarning(msg))

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/test_integration_server.py::test_stop_pipe - _queue.Empty
======= 1 failed, 628 passed, 2 skipped, 1 warning in 119.43s (0:01:59) ========</pre>
</details>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 18 packages built:</summary>
  <ul>
    <li>atomic-operator</li>
    <li>atomic-operator.dist</li>
    <li>ilspycmd</li>
    <li>knowsmore</li>
    <li>knowsmore.dist</li>
    <li>netexec</li>
    <li>netexec.dist</li>
    <li>powershell</li>
    <li>powershell-editor-services</li>
    <li>python312Packages.psrpcore</li>
    <li>python312Packages.psrpcore.dist</li>
    <li>python312Packages.pypsrp</li>
    <li>python312Packages.pypsrp.dist</li>
    <li>python313Packages.psrpcore</li>
    <li>python313Packages.psrpcore.dist</li>
    <li>python313Packages.pypsrp</li>
    <li>python313Packages.pypsrp.dist</li>
    <li>windmill</li>
  </ul>
</details>

---
### `x86_64-darwin` (sandbox = true)
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>ilspycmd</li>
    <li>powershell-editor-services</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>powershell</li>
  </ul>
</details>

---
### `aarch64-darwin` (sandbox = true)
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>ilspycmd</li>
    <li>powershell-editor-services</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>powershell</li>
  </ul>
</details>

--- a/maintainers/maintainer-list.nix
+++ b/maintainers/maintainer-list.nix
@@ -24187,6 +24187,14 @@
     githubId = 1552853;
     name = "Vincent Ambo";
   };
+  tbaldwin = {
+    email = "trent.baldwin@proton.me";
+    matrix = "@tbaldwin:matrix.org";
+    github = "tbaldwin-dev";
+    githubId = 220447215;
+    name = "Trent Baldwin";
+    keys = [ { fingerprint = "930C 3A61 F911 1296 7DA5  56D1 665A 9E2A FCDD 68AA"; } ];
+  };
   tbenst = {
     email = "nix@tylerbenster.com";
     github = "tbenst";

--- a/pkgs/by-name/po/powershell-editor-services/package.nix
+++ b/pkgs/by-name/po/powershell-editor-services/package.nix
@@ -7,11 +7,11 @@
 }:
 stdenvNoCC.mkDerivation rec {
   pname = "powershell-editor-services";
-  version = "4.3.0";
+  version = "4.4.0";
 
   src = fetchzip {
     url = "https://github.com/PowerShell/PowerShellEditorServices/releases/download/v${version}/PowerShellEditorServices.zip";
-    hash = "sha256-YiXxztfsYujJK6+56Lwi4QdRIZb7NH+dAUJTOYk8pGI=";
+    hash = "sha256-tmz6xkcF44xnG5VwSeHNWzjRQK6wxQ7Ii6mDNisCMb8=";
     stripRoot = false;
   };
 

--- a/pkgs/development/tools/ilspycmd/default.nix
+++ b/pkgs/development/tools/ilspycmd/default.nix
@@ -8,15 +8,15 @@
   autoSignDarwinBinariesHook,
   glibcLocales,
 }:
-buildDotnetModule (finalAttrs: rec {
+buildDotnetModule (finalAttrs: {
   pname = "ilspycmd";
-  version = "9.0-preview3";
+  version = "9.1";
 
   src = fetchFromGitHub {
     owner = "icsharpcode";
     repo = "ILSpy";
-    rev = "v${version}";
-    hash = "sha256-7cPXFaEKr76GtqcNsKx7tstRUeTpSTF8ggxbyEnQa9M=";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-YkZEStCI6Omu8HgClm5qHnXxm5pKJVILtbydY8vAFic=";
   };
 
   nativeBuildInputs = [
@@ -53,6 +53,9 @@ buildDotnetModule (finalAttrs: rec {
       fromSource
       binaryBytecode
     ];
-    maintainers = with lib.maintainers; [ emilytrau ];
+    maintainers = with lib.maintainers; [
+      emilytrau
+      tbaldwin
+    ];
   };
 })

